app/Notifications/NewUnfollow.php

```diff
@@ -42,6 +42,24 @@ public function via(object $notifiable): array
         return ['database'];
     }
 
+    /**
+     * Determine if the notification should be sent.
+     */
+    public function shouldSend(User $notifiable): bool
+    {
+        if ($notifiable->notification?->block_notifications == 1) {
+            return false;
+        }
+
+        if (!$notifiable->notification?->show_account_unfollow) {
+            return false;
+        }
+
+        // If the sender's group ID is found in the "Block all notifications from the selected groups" array,
+        // the expression will return false.
+        return ! \in_array($this->unfollower->group_id, $notifiable->notification->json_following_groups, true);
```

Comment:
Should be `json_account_groups` to match the above and the settings page.

Reply:
Corrected.